shishito/runtime/environment/node_webkit.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+from selenium import webdriver
+import os
+import sys
+from shishito.runtime.environment.shishito import ShishitoEnvironment
+
+
+class ControlEnvironment(ShishitoEnvironment):
+    """ Node-webkit control environment. """
+
+    def call_browser(self, config_section):
+        """ Start chromedriver for node-webkit application. Chromedriver have to be located in same
+        folder as node-webkit application (AUT).
+
+        :param str config_section: section in platform/environment.properties config
+        :return: created webdriver
+        """
+        capabilities = ''
+        node_webkit = self.shishito_support.get_opt('app_path')
```

Comment:
You're getting node webkit path from `app_path` variable. But you're passing it from command line as `--node_webkit_path`.

Those names should be the same (unless I am overlooking something), otherwise the override within the `get_opt` method will not work - when cmd argument specified, it should be used, when not specified, value from config (defined by same name) should be used.

I suggest renaming it here to `self.shishito_support.get_opt('node_webkit_path')`